### DISCUSSION TITLE
New Single Post Page Design

### DISCUSSION THREAD

**Discussion by its-kunal** [keploy/keploy#1667] [Ideas]
Hey folks, There is a inconsistent layout for single post page.
so, I'm designing a layout for single post page. 
[Figma Link](https://www.figma.com/file/RcgKR6bQpMSFJAGMWEly5J/Keploy?type=design&node-id=0%3A1&mode=design&t=qXlQ6DIkpIYSH1bC-1)
Please suggest changes and references.

**Comment by jaiakash** [comment]
I cant access the figma, Can you allow public access to figma file?

**Comment by its-kunal** [comment]
@jaiakash, Allowed public access.

**Comment by its-kunal** [comment]
Hey @nehagup , do you have any suggestions for the layout of single post page

**Comment by nehagup** [comment]
Hi @its-kunal thanks for spending the time. I feel the space in the author's name and the date is too much, you can take reference from hashnode design https://keploy-blog.hashnode.dev/mongodb-in-mock-mode-acting-the-server-part#heading-beyond-binaries-parsing-packets-for-human-friendly-mocks 

The title should be center aligned and then the banner image and then the author details, category and date.

**Comment by its-kunal** [comment]
Thanks for informative review @nehagup, I will update the design.

**Comment by its-kunal** [comment]
Hey @nehagup i have done the changes similar to the given reference.

**Comment by its-kunal** [comment]
Hey @nehagup, any suggestions on the [design](https://www.figma.com/file/RcgKR6bQpMSFJAGMWEly5J/Keploy?type=design&node-id=0%3A1&mode=design&t=qXlQ6DIkpIYSH1bC-1) or can I start working on this single post page considering design references from hashnode blog?
